Source/WebInspectorUI/UserInterface/Base/Main.js

```diff
@@ -233,9 +233,6 @@ WI.contentLoaded = function()
 
     // Add platform style classes so the UI can be tweaked per-platform.
     document.body.classList.add(WI.Platform.name + "-platform");
-    if (WI.Platform.version.name)
-        document.body.classList.add(WI.Platform.version.name);
```

Comment:
it may be worth keeping this if we plan to keep `InspectorFrontendHost.platformVersionName` around so that in the future if things are added to it then we don't have to re-add this code